src/misc/KeyManager.ts

```diff
@@ -184,7 +183,8 @@ class KeyManager {
 	}
 
 	registerShortcuts(shortcuts: ReadonlyArray<Shortcut>) {
-		Keys.META.code = client.browser === BrowserType.FIREFOX ? 224 : 91
+		//FIXME~ shouldn't need this anymore?
+		//Keys.META.code = client.browser === BrowserType.FIREFOX ? 224 : 91
```

Comment:
if we can get the Meta key (Ctrl/Cmd I think?) in a platform-independent way now, it's indeed obsolete.